share/shaders/FramebufferND.frag

```diff
@@ -0,0 +1,24 @@
+#version 330 core
```

Comment:
What does this fragment shader do?

Reply:
It draws a framebuffer while translating OSPRay's depth buffer to OpenGL's depth buffer.

I added a comment.